ios/FlowerAI.xcodeproj/project.pbxproj

```diff
@@ -659,6 +676,24 @@
 			name = Products;
 			sourceTree = "<group>";
 		};
+		19F2676D1F9F9A34003723B9 /* Recovered References */ = {
```

Comment:
`Recovered References` is unnecessary info. So, I think we should remove the `Recovered References` group